src/common/components/cards/card-kebab-menu-button.tsx

```diff
@@ -0,0 +1,87 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import { IPoint } from '@uifabric/utilities';
+import { ActionButton } from 'office-ui-fabric-react/lib/Button';
+import { ContextualMenu, IContextualMenuItem } from 'office-ui-fabric-react/lib/ContextualMenu';
+import * as React from 'react';
+import { kebabMenuButton } from './card-footer.scss';
+
+export interface CardKebabMenuButtonState {
+    isContextMenuVisible: boolean;
+    target?: HTMLElement | string | MouseEvent | IPoint | null;
+}
+
+export interface CardKebabMenuButtonProps {}
+
+export class CardKebabMenuButton extends React.Component<CardKebabMenuButtonProps, CardKebabMenuButtonState> {
+    constructor(props: CardKebabMenuButtonProps) {
+        super(props);
+
+        this.state = {
+            isContextMenuVisible: false,
+        };
+    }
+
+    public render(): JSX.Element {
+        return (
+            <div className={kebabMenuButton}>
+                <ActionButton
+                    iconProps={{
+                        iconName: 'moreVertical',
+                    }}
+                    onClick={this.openDropdown}
+                />
+                {this.renderContextMenu()}
+            </div>
+        );
+    }
+
+    private renderContextMenu(): JSX.Element {
+        if (!this.state.isContextMenuVisible) {
+            return null;
+        }
+
+        return <ContextualMenu onDismiss={() => this.dismissDropdown()} target={this.state.target} items={this.getMenuItems()} />;
+    }
+
+    private getMenuItems(): IContextualMenuItem[] {
+        const items: IContextualMenuItem[] = [
+            {
+                key: 'fileissue',
+                name: 'File issue',
+                iconProps: {
+                    iconName: 'copy',
+                },
+                onClick: this.fileIssue,
+            },
+            {
+                key: 'copyfailuredetails',
+                name: `Copy failure details`,
+                iconProps: {
+                    iconName: 'ladybugSolid',
+                },
+                onClick: this.copyFailureDetails,
```

Comment:
The `copy` icon would make more sense to go with "copy failure details", and the `ladybugSolid` icon should go with "file issue".